src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -12,7 +12,7 @@ import {
 } from 'components';
 import { USER_ROLE_ADMIN, USER_ROLE_COORDINAPE } from 'config/constants';
 import { useNavigation } from 'hooks';
-import { useContributions } from 'hooks/useContributions';
+import { useDeprecatedContributions as useContributions } from 'hooks/useContributions';
```

Comment:
can we just ditch the deprecated one?

Reply:
or if not, use the actual deprecated name so that we know it should be used with caution when reading the code

Reply:
done